docs/src/starknet/show_config.md

```diff
@@ -0,0 +1,30 @@
+# Printing Current Configuration
+
+## Overview
+
+Many times before executing any other cast command you want to make sure that you are passing the right
+configuration settings and not mixing up commands that needs to be sent to development environment to mainnet :)
+
+For those cases you can call `show-config` with all the same configuration related arguments, you can also just
+replace subcommand and anything after it with `show-config` and it should show all the configuration that will be
+used.
```

Comment:
```suggestion
To see just that, a `show-config` subcommand  can be used. You can just
replace any subcommand (and its parameters) with `show-config` and it will show currently used configuration.
```